docs/src/fs.rst

```diff
@@ -161,7 +161,7 @@ API
     Cleanup request. Must be called after a request is finished to deallocate
     any memory libuv might have allocated.
 
-.. c:function:: int uv_fs_close(uv_loop_t* loop, uv_fs_t* req, uv_file file, uv_fs_cb cb)
+.. c:function:: int uv_fs_close(uv_loop_t* loop, uv_fs_t* req, uv_os_fd_t file, uv_fs_cb cb)
```

Comment:
I've added a `fixup!` commit to address the doc comments, assuming for now that I should go with `v2.0.0`.